pytest/conftest.py

```diff
@@ -78,7 +78,7 @@ def base_sample_resource(username='admin', title=str(uuid.uuid4()), contributor=
         owner=user,
         title=title,
         metadata=metadata,
-        files=(open('pytest.ini', 'rb'),)
+        # files=(open('pytest.ini', 'rb'),)
```

Comment:
Are you sure files parameter is not needed any more? If so, it is better to delete it rather than commenting it out.

Reply:
This line (or a more reliable file name/location across laptops and within the Docker container) would be if a test needed to perform an operation on a file. As none such exist, this is being removed from the original boilerplate that was copied over. There exist environments across CUAHSI assets that this pytest.ini does not exist and would crash the tests, based on this fixture failure. I left the code commented to save the next person time, as it may not be immediately obvious that files can be included in testing this way. Perhaps I should rename 'pytest.ini' to 'consistent_file_across_laptops_and_containers.txt' in the comment, though.

Reply:
OK, makes sense. I think "pytest.ini" file name is good. A comment to indicate this for the next person's reference would be better than commenting out the line in my opinion.

Reply:
I left it more as an action item for myself, since nobody else seems to be embracing the PyTest stuff. pytest.ini is an existing file, but only in the majority of situations, not every situation. Something more robust or created by the test setup needs to be created and I didn't want to do this as a part of Citations, but wanted to not fix this, since Dan has been working on Jenkins/PyTest automation and it may fail if I don't do this.